File: myapp/parent/Step10_Lambda.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.parent;

public class Step10_Lambda : SfnLambda<ParentPayload> {
  public override ParentPayload Execute(ParentPayload payload) {
    payload.HappyPathData += " > 10";
    return payload;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/Step10_Lambda.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step10_Lambda extends SfnLambda<ParentPayload> {
    @Override
    public ParentPayload execute(ParentPayload payload) {
        payload.setHappyPathData(payload.getHappyPathData() + " > 10");
        return payload;
    }
}
